bubblewrap.c

```diff
@@ -2162,7 +2168,7 @@ main (int    argc,
 
   __debug__ (("forking for child\n"));
 
-  if (opt_unshare_pid || lock_files != NULL || opt_sync_fd != -1)
+  if (!no_reaper && (opt_unshare_pid || lock_files != NULL || opt_sync_fd != -1))
```

Comment:
True, i was just looking at the handle_die_with_parent call in do_init(). It should work.